healthCheck/README.md

```diff
@@ -0,0 +1,85 @@
+# Health Check Configuration Guide
+
+This project uses the `elevate-project-services-health-check` package to perform health checks for internal components like MongoDB, Kafka, and dependent microservices.
+
+To enable this, create a configuration file (`health.config.js`) that defines what to check and how.
+
+---
+
+## ✅ Sample Configuration
+
+```js
+module.exports = {
+	name: 'ProjectService', // 🔹 Service name shown in health check response
+	version: '1.0.0', // 🔹 Service version shown in response
+
+	checks: {
+		mongodb: {
+			enabled: true, // ✅ Required if MongoDB is used
+			url: process.env.MONGODB_URL, // 🔐 Recommended: use env variable
+		},
+		gotenberg: {
```

Comment:
@MallanagoudaB the read me should not this specific.it should be general explaining what each key do and what is the criticality of it, just like how we have env sample . looking at the readme one should be able to  create a config. it is expected to be  that helpful